src/Records/Types/DomainName.php

```diff
@@ -22,7 +22,7 @@
  */
 class DomainName extends Type
 {
-    const FLAG_NO_COMPRESSION = 0x80000000;
+    const FLAG_NO_COMPRESSION = 0x8000;
```

Comment:
Technically, this is a BC break. Should we just fix the usage and leave this as is?

Reply:
This line is the culprit, php triggers the deprecation on this very line. It's a too big number on 32bit.

Reply:
See #28, WDYT?